internal/artifacts/module_arch.go

```diff
@@ -250,7 +250,7 @@ func handleCopyMtaContentFailure(targetLocation string, copiedMtaContents []stri
 	if errCleanup == nil {
 		return errors.Errorf(message, messageArguments...)
 	}
-	return errors.Errorf(message+cleanupFailedMsg, messageArguments...)
+	return errors.Errorf(message+";"+cleanupFailedMsg, messageArguments...)
```

Comment:
There should be a space after the `;`, shouldn't there?

Reply:
done